packages/shared/lib/models/Sync.ts

```diff
@@ -290,13 +290,15 @@ export interface SyncConfigWithProvider {
 export interface IntegrationServiceInterface {
     runScript(
         syncName: string,
+        syncId: string,
```

Comment:
`syncId` is an attribute of NangoSync I think

Reply:
Updated with https://github.com/NangoHQ/nango/pull/1364/commits/c3da33a3c16ade23b07c92c765565c6cf98f442e

construct an id for an action using the name, env id, provider config key, and connection id